integr8ly/cloud-resource-operator
# Openshift Basic Providers

## Overview

Currently we do support Openshift installs of blobstorage and smtp resources. This change adds the support and e2e 

## Verification

- Clone this branch
- Run `make cluster/prepare`
- Run `make run`
- Provision smtp and blobstorage workshops
- Ensure secrets are created
- Run `make cluster/clean`
- Run `make test/e2e/local`
- Ensure tests pass
